src/internal/m365/graph/concurrency_middleware.go

```diff
@@ -150,10 +150,12 @@ const (
 	// https://learn.microsoft.com/en-us/sharepoint/dev/general-development
 	// /how-to-avoid-getting-throttled-or-blocked-in-sharepoint-online#application-throttling
 	defaultLC      = 1
-	driveDefaultLC = 2
+	driveDefaultLC = 1
```

Comment:
if 1 is the default for drive, then do we need two consts?  This can probably get removed in favor of the single default.

Reply:
That works.